packages/backend/src/services/ProjectService/formatRows.js

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable */
+/* eslint-disable */
+const { workerData, parentPort } = require('worker_threads');
+
+const { formatItemValue } = require('@lightdash/common');
+
+function formatRows(rows, itemMap) {
+    const formattedRows = rows.map((row) =>
```

Comment:
Couldn't we just import the `formatRows` from common? 🤔 
We can update that function to receive the itemMap as we did here.

Reply:
Nice suggestion, done